packages/opentelemetry-exporter-collector/src/platform/node/sendSpans.ts

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const http = require('http');
+const https = require('https');
+
+import { IncomingMessage } from 'http';
+import { hrTime, hrTimeToTimeStamp } from '@opentelemetry/core';
+import { CollectorExporter } from '../../CollectorExporter';
+
+import * as collectorTypes from '../../types';
+
+const url = require('url');
+
+/**
+ * function that is called once when {@link ExporterCollector} is initialised
+ * in node version this is not used
+ * @param shutdownF shutdown method of {@link ExporterCollector}
+ */
+export function onInit(shutdownF: Function) {}
```

Comment:
should we register a process shutdown handler here? Possibly by listening for `beforeExit` event on `process`?